tutorials/epochs/make_fixed_length_epochs.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-fixed-length-epochs:
+
+Creating epochs of equal length
+===============================
+
+This tutorial shows how to create equal length epochs and briefly demonstrates
+an example of their use in connectivity analysis.
+
+First, we import necessary modules and read in a sample raw
+data set. This data set contains brain activity that is event-related, i.e.
+synchronized to the onset of auditory stimuli. However, rather than creating
+epochs by segmenting the data around the onset of each stimulus, we will
+create 30 second epochs that allow us to perform non-event-related analyses of
+the signal.
+"""
+
+import os
+import mne
+import matplotlib.pyplot as plt
+from mne.preprocessing import compute_proj_ecg
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+
```

Comment:
```suggestion
import os
import numpy as np
import mne
from mne.preprocessing import compute_proj_ecg
import matplotlib.pyplot as plt
from mpl_toolkits.axes_grid1 import make_axes_locatable

```

Reply:
Thanks, @larsoner and @agramfort for the reviews! Alex, what is the significance of the import reordering?

Reply:
it should reflect the hierarchy of dependencies between packages. First you put the packages from the standard library, then numpy, then matplotlib as it depends on numpy, then mne as it depends on the above.

it's not necessary but it's useful convention.